src/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,41 @@
+import { ProfileInterface } from "@/interfaces/ProfileInterface";
+import {
+  ProfileContainer,
+  ProfileContent,
+  ProfileLinks,
+  ProfileDescription,
+  ProfileImage,
+  ProfileName,
+} from "./Profile.styles";
+
+const Profile = (data: ProfileInterface) => {
+  return (
```

Comment:
Instead of using "data.<field" we could destructure the given data prop link this:

 ```suggestion
 const { name, photo, role, github, linkedin, lattes } = data
 
  return (
```